torch/ao/quantization/_dbr/utils.py

```diff
@@ -281,27 +281,27 @@ class FuncOutputDTypeType(enum.Enum):
     DTYPE_DEFAULT_BC_UNSUPPORTED_SYNTAX = 2
 
 def get_func_output_dtype_type(
-    op: Callable,
-    args: Tuple[Any, ...],
-    op_packing_only_uses_module_attributes: bool,
+    seen_op_info: SeenOpInfo,
```

Comment:
This part is just clean up right? (not strictly needed for function fusion?)

Reply:
correct, this is a refactor (after code was moved around, it was easier to just pass `seen_op_info` here which contains all the metadata instead of individual metadata instances)